x/staking/src/proto.rs

```diff
@@ -45,7 +78,7 @@ pub struct Validator {
     /// unbonding_height defines, if unbonding, the height at which this validator has begun unbonding.
     pub unbonding_height: i64,
     /// unbonding_time defines, if unbonding, the min time for the validator to complete unbonding.
-    pub unbonding_time: std::time::Duration,
+    pub unbonding_time: chrono::DateTime<Utc>,
```

Comment:
`pub consensus_pubkey: PublicKey,` above, is there a reason why this isn't a Tendermint public key?

Reply:
I had no particular reason to use the domain `PublicKey` instead of tendermint `PublicKey`. I suggested that it would be better to use domain types and used it. 

Should I change the type to tendermint `PublicKey` from https://github.com/rumos-io/gears/blob/main/tendermint/src/types/proto/crypto.rs#L2?

Reply:
The application keys (the ones in Gears) should be separate to the consensus keys (tendermint). You're right we should generally use domain types. I suggest we create a domain type for the tendermint pub key in `gears::tendermint.`